ldi-orchestrator/ldio-connectors/ldio-azure-blob-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/conversionstrategy/ConversionStrategyFactory.java

```diff
@@ -0,0 +1,20 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.conversionstrategy;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldio.exception.UnsupportedLangException;
+import org.apache.jena.riot.Lang;
+
+import static org.apache.jena.riot.RDFLanguages.nameToLang;
+
+public class ConversionStrategyFactory {
+
+	public ConversionStrategy getConversionStrategy(String outputLanguage, String jsonContextURI) {
+		if (outputLanguage.equals("json")) {
```

Comment:
Can we use full mimetype here? application/json

Reply:
I'm a bit in dubio here, because mimetype is something from the HTTP context. Here, we're rather looking at the file type/file extension. E.g. you could also convert it to a csv or a pdf or a ... and they don't really have mime-types.

Reply:
text/csv, application/pdf ? :) https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types